kuma/dashboards/views.py

```diff
@@ -69,16 +70,12 @@ def revisions(request):
 
     if query_kwargs:
         revisions = revisions.filter(**query_kwargs)
-        total = revisions.count()
-    else:
-        # If no filters, just do a straight count(). It's the same
-        # result, but much faster to compute.
-        total = Revision.objects.count()
 
     # Only bother with this if we're actually going to get
```

Comment:
Here's where the comment was added: https://github.com/mozilla/kuma/commit/39dd99f487e6c503e5dcddeb331f4121348c44b2

ubernostrum was attempting to short-circuit pagination if the requested page was past the total number of pages.  This was back when the revisions dashboard was implemented with JSON, rather than Django pagination.  The code in question is no longer executed, so there is no reason to efficiently count the number of revisions, filtered or not.

Django pagination calls ``COUNT(*)`` to get the total, and does its own work to validate page numbers.  The total item count is available from the page object.  I think adding total to the context is definitely a artifact from the old way of paginating this page, and should be removed in favor of the changes in https://github.com/mozilla/kuma/commit/167fe92b6b9c